services/frontend-v3/src/pages/admin/User.jsx

```diff
@@ -3,20 +3,16 @@ import AdminLayout from "../../components/layouts/adminLayout/AdminLayout";
 import UserTable from "../../components/userTable/UserTable";
 import { injectIntl } from "react-intl";
 
-class AdminUser extends React.Component {
-  goto = link => this.props.history.push(link);
-
-  render() {
-    return (
-      <AdminLayout
-        changeLanguage={this.props.changeLanguage}
-        displaySideBar={true}
-        type="user"
-      >
-        <UserTable type="user" />
-      </AdminLayout>
-    );
-  }
+function AdminUser(props) {
+  return (
+    <AdminLayout
+      changeLanguage={props.changeLanguage}
+      displaySideBar={true}
+      type="user"
+    >
+      <UserTable type="user" />
+    </AdminLayout>
+  );
 }
 
 export default injectIntl(AdminUser);
```

Comment:
remove injectint if you dont need it. please check everywhere for this

Reply:
Fixed.